### PR TITLE
Update email address used for package maintainer

### PR DESCRIPTION
**Pull request by thomasspriggs**
Emails sent to this new email address should reach the group responsible
for cbmc maintenance more quickly.

- [x] Each commit message has a non-empty body, explaining why the change was made.
- [ ] N/A - Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] N/A - The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] N/A - Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] N/A - My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- [x] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Comment by thomasspriggs** [comment]
@peterschrammel Do you think we should also replace other usages of the info@diffblue.com email addresses?

https://github.com/diffblue/cbmc/blob/develop/src/symtab2gb/symtab2gb_parse_options.cpp#L124
https://github.com/diffblue/cbmc/blob/develop/src/memory-analyzer/memory_analyzer_parse_options.cpp#L174

**Review by NlightNFotis — APPROVED** [review]

File: cmake/packaging.cmake [hunk 1]
```diff
@@ -1,6 +1,6 @@
 set(CPACK_PACKAGE_NAME "cbmc")
 set(CPACK_PACKAGE_VENDOR "Diffblue Ltd.")
-set(CPACK_PACKAGE_CONTACT "info@diffblue.com")
+set(CPACK_PACKAGE_CONTACT "Diffblue open source team <cbmc@diffblue.com>")
```
Comment (hannes-steffenhagen-diffblue) [inline cmake/packaging.cmake]: :pick: I believe Open Source Team should be capitalised since it's a name?

**Review by hannes-steffenhagen-diffblue — APPROVED** [review]
LGTM

**Review by martin-cs — APPROVED** [review]
Up to date contact information is always good.

**Comment by martin-cs** [comment]
CI seems to have stalled.  Any chance you could kick it, then I will merge.

**Pull request merged**